src/test/java/com/navfort/step_definitions/US50_MainModulesStepDefs.java

```diff
@@ -0,0 +1,36 @@
+package com.navfort.step_definitions;
+
+import com.navfort.pages.LoginPage;
+import com.navfort.pages.MainModulesPage;
+import com.navfort.utilities.BrowserUtils;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+
+import java.util.List;
+
+public class US50_MainModulesStepDefs {
+
+    LoginPage loginPage = new LoginPage();
+
+    @When("user enters {string} information")
+    public void userEntersInformation(String arg0) {
```

Comment:
Hi Mokhinur. You actually don't need to do login part. Because there is a ready login test step and method for that. You need to provide only the user type (like: driver, sales manager or store manager).

Reply:
Yes but I copied login part from ready file, am I wrong?